querybook/webapp/components/DataElement/DataElementCard.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+import { IDataElement } from 'const/dataElement';
+import { DataElementResource } from 'resource/table';
+import { IconButton } from 'ui/Button/IconButton';
+import { KeyContentDisplay } from 'ui/KeyContentDisplay/KeyContentDisplay';
+import { AccentText } from 'ui/StyledText/StyledText';
+
+import './DataElement.scss';
+
+interface IProps {
+    dataElement: IDataElement;
+}
+
+export const DataElementCard = ({ dataElement }: IProps) => {
+    const { id, name, type, description, properties } = dataElement;
+
+    const onExternalLinkClick = async () => {
```

Comment:
useCallback

Reply:
As we discussed this before, other than cases like 
 - it will be depended by other hooks
  - it's a true performance issue
 the cost of creating the memoized function itself may not less than the cost of creating the function itself. So i'd prefer not using `useCallback` if not really needed.